.github/workflows/deploy-docs.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     - name: Before install
       run: |
         sudo apt-get update
-        sudo apt-get install libgdal-dev graphviz
+        sudo apt-get install libgdal-dev graphviz graphviz-dev
```

Comment:
Does this line need both `graphviz` and `graphviz-dev` or is just `graphviz-dev` sufficient?

Reply:
We need both.  I just checked and 
```
sudo apt-get install graphviz-dev
```
doesn't install ``graphviz``, which is needed since ``pygraphviz`` calls the commandline tools.